EpicCactus94.md

```diff
@@ -0,0 +1,38 @@
+name: "@EpicCactus94"
+project: "Self Balancing Reaction Wheel Cube"
+---
+
+# Project Name: Self Balancing Reaction Wheel Cube
+
+## Summary
+
+Design and 3D print a reaction wheel cube and code it to be able to balance on a corner (even on inclined surfaces) and react to disturbances 
+like pushing. I was inspired to build this project from videos online of how they're used in satelites.
+
+## Plan
+
+I've already begun designing the .stl files which will be used to print the cube, and I've assured access to 3D printers before the building week. 
```

Comment:
Fascinating! Could you describe the process of creating these 3D models? What CAD tools are you using / what is your design process like?